packages/io/dxf-serializer/index.js

```diff
@@ -405,6 +431,34 @@ const getEntityId = (options) => {
   return 'CAD' + padded.substr(padded.length - 5)
 }
 
+const getName = (object, options) => {
+  if (object.name) return object.name
+  // add more zeros if the id needs to be larger
+  const padded = '00000' + options.entityId.toString(16).toUpperCase()
+  return 'CAD' + padded.substr(padded.length - 5)
+}
+
+const getColorNumber = (object, options) => {
```

Comment:
DXF color table seems to be organized into the most used colors first. It's probably worth exiting the loop if diff == 0, as most calls will then not need to iterate over all 256 entries.

Reply:
Nice suggestion. I’ll make a small change..